addons/crm_claim/crm_claim.py

```diff
@@ -38,144 +18,102 @@ class crm_claim_stage(osv.osv):
-        'cause': fields.text('Root Cause'),
-    }
-
-    _defaults = {
-        'user_id': lambda s, cr, uid, c: uid,
-        'team_id': lambda s, cr, uid, c: s._get_default_team_id(cr, uid, c),
-        'date': fields.datetime.now,
-        'company_id': lambda s, cr, uid, c: s.pool.get('res.company')._company_default_get(cr, uid, 'crm.case', context=c),
-        'priority': '1',
-        'active': lambda *a: 1,
-        'stage_id': lambda s, cr, uid, c: s._get_default_stage_id(cr, uid, c)
-    }
-
-    def stage_find(self, cr, uid, cases, team_id, domain=[], order='sequence', context=None):
+    _inherit = 'mail.thread'
+
+    name = fields.Char('Claim Subject', required=True)
+    active = fields.Boolean(default=True)
+    action_next = fields.Char('Next Action')
+    date_action_next = fields.Datetime('Next Action Date')
+    description = fields.Text()
+    resolution = fields.Text()
+    create_date = fields.Datetime('Creation Date', readonly=True)
+    write_date = fields.Datetime('Update Date', readonly=True)
+    date_deadline = fields.Date('Deadline')
+    date_closed = fields.Datetime('Closed', readonly=True)
+    date = fields.Datetime('Claim Date', select=True, default=fields.datetime.now())
+    ref = fields.Selection(lambda s: referencable_models(s, s._cr, s._uid, s._context), 'Reference')
+    categ_id = fields.Many2one('crm.claim.category', 'Category')
+    priority = fields.Selection([('0', 'Low'), ('1', 'Normal'), ('2', 'High')], default='1')
+    type_action = fields.Selection([('correction', 'Corrective Action'), ('prevention', 'Preventive Action')], 'Action Type')
+    user_id = fields.Many2one('res.users', 'Responsible', track_visibility='always', default=lambda s: s._uid)
+    user_fault = fields.Char('Trouble Responsible')
+    team_id = fields.Many2one(
+        'crm.team', 'Sales Team', oldname='section_id', select=True,
+        default=lambda s: s.env['crm.lead']._resolve_team_id_from_context() or False,
+        help="Responsible sales team. Define Responsible user and Email account for mail gateway.")
+    company_id = fields.Many2one('res.company', 'Company', default=lambda s: s.env['res.company']._company_default_get('crm.case'))
+    partner_id = fields.Many2one('res.partner', 'Partner')
+    email_cc = fields.Text(
+        'Watchers Emails', size=252,
+        help="These email addresses will be added to the CC field of all inbound and outbound emails for this record before being sent. \
+        Separate multiple email addresses with a comma")
+    email_from = fields.Char('Email', size=128, help="Destination email for email gateway.")
+    partner_phone = fields.Char('Phone')
+    stage_id = fields.Many2one(
+        'crm.claim.stage', 'Stage', track_visibility='onchange',
+        default=lambda s: s.stage_find([], s.team_id, [('sequence', '=', '1')]),
+        domain="['|', ('team_ids', '=', team_id), ('case_default', '=', True)]")
```

Comment:
use domain without string in py side .like domain=['|', ('team_ids', '=', team_id), ('case_default', '=', True)]